computation_graph/composers.py

```diff
@@ -228,7 +237,7 @@ def _infer_composition_edges(
 
 
 def make_compose(
-    *funcs: _ComposersInputType, key: Optional[Text] = None
+    *funcs: _ComposersInputType, key: Optional[Text] = None, is_future=False
```

Comment:
can we not add a default ?

i think you can change this into `_make_compose_inner(*funs, key, is_future)`

and then have `make_compose` (which will set is_future to False)
and add a `make_compose_future()`